contracts/src/v0.8/dev/ocr2dr/OCR2DROracle.sol

```diff
@@ -108,6 +108,7 @@ contract OCR2DROracle is OCR2DROracleInterface, OCR2Base, AuthorizedOriginReceiv
     bytes calldata data,
     uint32 gasLimit,
     uint256 gasPrice
+  // Is it necessary to use gas to check if the registry is set?  If it isn't set, the tx will fail anway.
```

Comment:
The current way gives an error message. So it's a trade-off better UX vs saving a few gas.

Reply:
Fair point, but my assumption is that we would have a single deploy script, so a user would never encounter this issue.

However I would guess the gas use here is trivial & this comment can be ignored.